src/dvc_studio_client/post_live_metrics.py

```diff
@@ -123,6 +123,8 @@ def post_live_metrics(  # noqa: C901,PLR0912,PLR0913
     studio_token: Optional[str] = None,
     studio_repo_url: Optional[str] = None,
     studio_url: Optional[str] = None,
+    dvc_experiment_parent_data: Optional[Dict[str, Any]] = None,
+    subdir: Optional[str] = None,
```

Comment:
Can we split `subdir` into its own PR?

Reply:
Yes, I was hoping to solve the detached experiments problem yesterday and not have to go around the loop twice.

I will repurpose/redo all of the PRs.